packages/components/src/FormField/FormField.tsx

```diff
@@ -262,6 +268,7 @@ export function FormField({
       readOnly: readonly,
       onChange: handleChange,
       value: value,
+      ...(keyboard && { inputMode: keyboard }),
```

Comment:
I believe that this can just be `inputMode: keyboard`.

If a prop is `undefined` it wont be added to the DOM.

Reply:
oh cool, didn't know that